### PR TITLE
ENH backport gcc930 to 3.13

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{% set version = "3.14.0" %}
-{% set sha256 = "a8f9caba03e0d57d8452c08505cf96be5f6949adaa266e819382162c03ddb9c5" %}
+{% set version = "3.13.5" %}
```
Comment (dalcinl) [inline recipe/meta.yaml:1]: Why not `3.13.6` ?? That's the last patch release of the 3.13 release series.

Comment (beckermr) [reply recipe/meta.yaml:1]: Sure! I grabbed the latest commit I could find with 3.13.x

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-{% set version = "3.14.0" %}
-{% set sha256 = "a8f9caba03e0d57d8452c08505cf96be5f6949adaa266e819382162c03ddb9c5" %}
+{% set version = "3.13.6" %}
+{% set sha256 = "10fc542dab961c8b17db35ad3a208cb184c237fc84e183817e38e6c7ab4b8732" %}
```
Comment (beckermr) [inline recipe/meta.yaml:2]:
```suggestion
{% set sha256 = "67ca2cf3040d08fdc51d27f660ea3157732b24c2f47aae1b19d63f62a39842c2" %}
```

**Comment by beckermr** [comment]
We may need to update the patches here. Let's see if it passes.

**Pull request merged**